app/services/message_employee_matcher.rb

```diff
@@ -21,10 +21,9 @@ def day_count
   end
 
   def match_time(time_zone)
-    employee_current_hour = Time.current.in_time_zone(time_zone).hour
-    employee_current_minute = Time.current.in_time_zone(time_zone).min
+    employee_current_time = Time.current.in_time_zone(time_zone)
 
-    employee_current_hour == message.time_of_day.hour && employee_current_minute == message.time_of_day.min
+    employee_current_time.utc.strftime("%H%M%S%N") >= message.time_of_day.utc.strftime("%H%M%S%N")
```

Comment:
What do you think about converting the dates/times to integers and comparing those instead of comparing strings?